src/packages/engine-core/src/undici.ts

```diff
@@ -1,18 +1,17 @@
 import getStream = require('get-stream')
-import { Pool } from 'undici'
-
+import { Client, Pool } from 'undici'
+import { URL } from 'url'
 export class Undici {
-  private pool: any
+  private pool: Pool
   private closed = false
-  constructor(url: any, moreArgs?: any) {
-    this.pool = new Pool(url, {
+  constructor(url: string | Partial<URL>, moreArgs?: Pool.Options) {
+    this.pool = new Pool(url as string, {
```

Comment:
Fix for this type as it should accept `string | URL` https://github.com/nodejs/undici/pull/507